.sass-lint.yml

```diff
@@ -0,0 +1,4 @@
+options:
+  max-warnings: 0
+files:
+  include: 'src/**/*.scss'
```

Comment:
is this needed? do we add any custom sass files?

Reply:
We don't, but we've got sass check in place